gpjax/gaussian_distribution.py

```diff
@@ -149,7 +152,9 @@ def entropy(self) -> ScalarFloat:
             self.event_shape[0] * (1.0 + jnp.log(2.0 * jnp.pi)) + self.scale.log_det()
         )
 
-    def log_prob(self, y: Float[Array, " N"]) -> ScalarFloat:
+    def log_prob(
```

Comment:
It would be great to add a test just for the masked case in `test_gaussian_distribution.py`. This way if something is off in this core functionality, one instantly knows.

Reply:
Yes, I have to add this and other tests.

Reply:
+1